src/content/import_service.cc

```diff
@@ -825,6 +825,10 @@ void ImportService::assignFanArt(
     }
 
     auto fanart = container->getResource(ResourcePurpose::Thumbnail);
+    // Ignore if the fanart's FanArtObject is not set
+    if (fanart && stoiString(fanart->getAttribute(ResourceAttribute::FANART_OBJ_ID)) <= CDS_ID_ROOT)
```

Comment:
You should add `fanart->getHandlerType() != ContentHandler::CONTAINERART` because ContainerArt does not set `FANART_OBJ_ID`